releasenotes/notes/allow-openai-client-config-4e1dc48f163912d0.yaml

```diff
@@ -0,0 +1,7 @@
+---
+highlights: >
+  AzureOpenAITextEmbedder now supports custom HTTP client configs via `custom_client_kargs`, enabling proxy and SSL setup.
+
+features:
+  - |
+    Added `custom_client_kargs` to AzureOpenAITextEmbedder for configuring HTTP client options like proxies and SSL.
```

Comment:
I think should only be a `feature`, it makes sense to mention both `AzureOpenAITextEmbedder` and `AzureOpenAIDocumentEmbedder`, and I would also keep it short and direct.

```suggestion
    `AzureOpenAITextEmbedder` and `AzureOpenAIDocumentEmbedder` now support custom HTTP client configs via `custom_client_kargs`, enabling proxy and SSL setup.
```